relay.go

```diff
@@ -229,9 +235,10 @@ func (r *Relayer) canHandleNewCall() bool {
 	return canHandle
 }
 
-func (r *Relayer) getDestination(f lazyCallReq) (*Connection, bool, error) {
+func (r *Relayer) getDestination(f lazyCallReq, cs relay.CallStats) (*Connection, bool, error) {
 	if _, ok := r.outbound.Get(f.Header.ID); ok {
 		r.logger.WithFields(LogField{"id", f.Header.ID}).Warn("received duplicate callReq")
+		cs.Failed(ErrCodeProtocol.MetricsKey())
```

Comment:
here and below, any errors that come out of the relay code should be prefixed with `relay-`